packages/vue/README.md

```diff
@@ -0,0 +1,131 @@
+# @synergy-design-system/vue
+
+This package provides [vue.js](https://vuejs.org/) wrappers for [Synergy Web Components](https://github.com/SickDesignSystem/synergy/tree/main/packages/components).
+
+This package aims to provide an improved developer experience in vue applications:
+
+- Provides two way data binding and `v-model`
+- Autocompletion and Types
+- Better custom event handling of `synergy` events
+
+## Getting started
+
+### 1. Package installation
+
+Run the following steps to install the required packages.
+
+```bash
+# Install the base library and required css files
+npm install --save @synergy-design-system/vue @synergy-design-system/tokens
+```
+
+> ⚠️ Note we do **not** ship vue in this package.
+> You will have to install it by yourself first!
+
+### 2. Add the wanted theme to your application
+
+The components will not display correctly without the needed theme. Please include either light or dark theme in your application, for example in a newly installed vue application:
+
+```ts
+// src/main.ts
+// Add this line to enable the light theme for your application
+import '@synergy-design-system/tokens/dist/themes/light.css';
```

Comment:
When I tested this in a vite vue project it complained about not finding the path to the light.css. Removing the dist worked:
![image](https://github.com/SickDesignSystem/synergy/assets/115139104/111a658e-665c-4f20-b7a8-c2b33887a992)